Source/missiles.h

```diff
@@ -34,6 +34,38 @@ struct MissilePosition {
 	Point traveled;
 };
 
+
+/*
+ *      W  sW  SW   Sw  S

```

Comment:
South is pointing up? What's going on here?

Reply:
Those are actually the x and y axes. If something is moving along the Y axis, it's going SW on-screen.

Reply:
As @AJenbo mentioned the order these values are declared matches the layout of missile sprite sheets (e.g. https://www.spriters-resource.com/pc_computer/diablodiablohellfire/sheet/97919/). With an isometric grid DIR16_S is pointing directly to the bottom of the screen and corresponds to movement along both x and y axes (+1, +1).